conda/gateways/repodata/__init__.py

```diff
@@ -471,15 +519,15 @@ def __setitem__(self, key: str, item: Any) -> None:
 
     def __missing__(self, key: str):
         if key in self._aliased:
-            key = key[1:]  # strip underscore
+            key = self._aliased[key]
         else:
             raise KeyError(key)
         return super().__getitem__(key)
 
 
 class RepodataCache:
     """
-    Handle caching for a single repodata.json + repodata.state.json
+    Handle caching for a single repodata.json + repodataa separate file
```

Comment:
> `repodataa separate file`

Similar issue as the above, I think there might have been a search + replace gone awry